dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Models/OpenAISdkChatMessageExensions.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Azure.AI.OpenAI;
+using Azure.AI.OpenAI.Chat;
+using Microsoft.Teams.AI.Exceptions;
+using OpenAI.Chat;
+using OAI = OpenAI;
+
+namespace Microsoft.Teams.AI.AI.Models
+{
+    /// <summary>
+    /// Provides extension methods for the <see cref="ChatCompletion"/> class.
+    /// </summary>
+    internal static class OpenAISdkChatMessageExensions
+    {
+        /// <summary>
+        /// Converts an <see cref="ChatCompletion"/> to a <see cref="ChatMessage"/>.
+        /// </summary>
+        /// <param name="chatCompletion">The original <see cref="OAI.Chat.ChatMessage" />.</param>
+        /// <returns>A <see cref="ChatMessage"/>.</returns>
+        public static ChatMessage ToChatMessage(this ChatCompletion chatCompletion)
```

Comment:
instead of this static class with helpers, I would consider adding new constructors to the classes you are trying to convert the type to, here is an example https://stackoverflow.com/questions/15251167/how-to-extend-a-class-in-c

Reply:
pushed fix